README.md

```diff
@@ -93,6 +95,19 @@ example, the following 3 invocations are identical:
 2. `sev-snp-measure --vcpu-sig=0x800f12 ...`
 3. `sev-snp-measure --vcpu-family=23 --vcpu-model=1 --vcpu-stepping=2 ...`
 
+## Precalculated OVMF hashes
+
+In situations where only minor OVMF changes happen, you may not want to copy the full OVMF binary to the validation system. In these situations, you can use an OVMF hash.
+To generate a hash, use the `--gen-ovmf-hash` parameter:
```

Comment:
I originally had `--gen-ovmf-hash` as a special argument instead of `--mode=ovmf-hash`. But while writing the README and seeing how clumsy everything became, I figured I'd fold it in with `--mode`.